index.html

```diff
@@ -8,28 +8,48 @@
     <title>Awesome Books</title>
     <link rel="stylesheet" href="style.css">
     <script defer src="script.js"></script>
+    <script defer src="pageDisplay.js"></script>
 </head>
 
 <body>
-    <section id="book-section">
-        <button id="show-books">show books</button>
-    </section>
+    <header>
+        <nav class="nav-bar">
+            <div>Awesome Books</div>
+            <ul class="links-list">
+                <li><a id="list" class="nav list" href="#book-section">List</a></li>
+                <li id="navAdd"><a class="nav add" href="#add-book">Add new</a></li>
+                <li id="navContact"><a class="nav contact" href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section id="book-section">
+            <h1 class="mainTitle">Awesome books</h1>
+            <ul id="book-list"></ul>
+        </section>
 
-    <section id="add-book">
-        <div>
-            <h1 class="awesome-title">Awesome Books</h1>
-        </div>
-        <form>
-            <br>
-            <input id="title" type="text" name="Title" placeholder="Book Title:">
-            <br>
-            <br>
-            <input id="author" type="text" name="Author" placeholder="Author:">
-            <br><br>
-            <button id="add">Add</button>
-        </form>
-    </section>
-  
+        <section id="add-book">
+            <h2 class="awesome-title mainTitle">Add a new book</h2>
+            <form>
+                <input id="title" type="text" name="Title" placeholder="Book Title:" required>
+                <input id="author" type="text" name="Author" placeholder="Author:" required>
+                <button id="add">Add</button>
+            </form>
+        </section>
+
+        <section id="contact">
+            <h2 class="mainTitle">Contact information</h2>
+            <p>Do you have any questions or you just want to say hello?</p>
+            <ul>
+                <li>Our email: mail@mail.com </li>
+                <li>Our phone number</li>
+                <li>Our address Streethome 22, 82433 City, Country</li>
+            </ul>
+        </section>
+    </main>
+    <footer>
+        <p class="footer">Copywrite</p>
+    </footer>
 </body>
```

Comment:
- [ ]  Your footer should be exact similar to the one from provided design and make sure that it will be fixed at the bottom of the page while you will be adding the tasks. 
![image](https://user-images.githubusercontent.com/43877689/177600600-4eab9649-964a-4c94-9e40-5f7b6cef5055.png)